_targets.R

```diff
@@ -0,0 +1,34 @@
+library(targets)
+source('code.R')
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
+    fetch_data(project_output_dir = "1_fetch/out/model_RMSEs.csv"),
+    format = "file"
+  ), 
+  # Prepare the data for plotting
+  tar_target(
+    eval_data,
+    process_data(project_input_dir = model_RMSEs_csv),
+  ),
+  # Create a plot
+  tar_target(
+    model_plot_png,
+    plot_data(project_output_dir = "3_visualize/out/figure_1.png", data = eval_data), 
```

Comment:
ditto with the `project_output_dir` variable name. This should be something more representative of what is being passed through, e.g. `out_file`